### ISSUE TITLE
ConfigTest.honorClientAuthenticatorCommands fails on Windows

### ISSUE THREAD

**Issue by pdudits** [fabric8io/kubernetes-client#3244]
The test fails with parsing error 

```
[main] ERROR io.fabric8.kubernetes.client.Config - Failed to parse the kubeconfig.
expected '<document start>', but found '<scalar>'
 in 'reader', line 1, column 5:
    '.' is not recognized as an internal ...
        ^
```
Because the output from `cmd /c ./token-generator-win` is `'.' is not recognized as an internal or external command,
operable program or batch file.`

The config file needs to use platform's path separator, which is backslash. 
Issue will be fixed alongside #3243

**Comment by rohanKanojia** [comment]
Does the client compile successfully on windows after fixing this test? Should we add a [GitHub windows workflow ](https://github.com/fabric8io/docker-maven-plugin/blob/master/.github/workflows/windows-build.yml)so that we don't break build on windows again in future?

**Comment by pdudits** [comment]
It needed one more fix that is present in #3245 but then `mvn clean install` passed.

**Comment by rohanKanojia** [comment]
@pdudits : Thanks a lot!
